core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1541,4 +1541,10 @@ package object config {
     .bytesConf(ByteUnit.BYTE)
     .createOptional
 
+  private[spark] val NATIVE_L1_THRESHOLD =
+    ConfigBuilder("spark.ml.blas.nativeL1Threshold")
+      .doc("vector size for logic change between f2jBLAS and natveBLAS in " +
```

Comment:
"Minimum vector size to use native BLAS libraries (if available) for Level 1 routines rather than Java implementation"
I'd also actually remove the "ml" from the property name as it isn't in an ML package, and isn't specific to ML. However we can document it in the context of ML as that is its only real use right now.